docs/source/API/containers/ScatterView.rst

```diff
@@ -0,0 +1,126 @@
+
+    .. cppkokkos:type:: internal_data_type
+
+        Value type of data_type_info.
+
+    .. cppkokkos:type:: internal_view_type
+
+        A View type created from the internal_data_type.
+
+    .. rubric:: Constructors
+
+    .. cppkokkos:function:: ScatterView()
+
+        The default constructor. Default constructs members.
+
+    .. cppkokkos:function:: ScatterView(View<RT, RP...> const&)
+
+        Constructor from a ``Kokkos::View``. ``internal_view`` member is copy constructed from this input view.
+
+    .. cppkokkos:function:: ScatterView(std::string const& name, Dims ... dims)
+
+        Constructor from variadic pack of dimension arguments. Constructs ``internal_view`` member.
+
+    .. cppkokkos:function:: ScatterView(::Kokkos::Impl::ViewCtorProp<P...> const& arg_prop, Dims... dims)
+
+        Constructor from variadic pack of dimension arguments. Constructs ``internal_view`` member. This constructor allows specifying an execution space instance to be used by passing, e.g. ``Kokkos::view_alloc(exec_space, "label")`` as first argument.
+
+    .. rubric:: Public Methods
+
+    .. cppkokkos:function:: constexpr bool is_allocated() const
+
+        :return: true if the ``internal_view`` points to a valid memory location. This function works for both managed and unmanaged views. With the unmanaged view, there is no guarantee that referenced address is valid, only that it is a non-null pointer.
+
+    .. cppkokkos:function:: access() const
+
+       use within a kernel to return a ``ScatterAccess`` member; this member accumulates a given thread's contribution to the reduction.
+
+    .. cppkokkos:function:: subview() const
+
+        :return: a subview of a ``ScatterView``
+
+    .. cppkokkos:function:: contribute_into(View<DT, RP...> const& dest) const
+
+       contribute ``ScatterView`` array's results into the input View ``dest``
+
+    .. cppkokkos:function:: reset()
+
+       performs reset on destination array
+
+    .. cppkokkos:function:: reset_except(View<DT, RP...> const& view)
+
+       tbd
+
+    .. cppkokkos:function:: resize(const size_t n0 = 0, const size_t n1 = 0, const size_t n2 = 0, const size_t n3 = 0, const size_t n4 = 0, const size_t n5 = 0, const size_t n6 = 0, const size_t n7 = 0)
+
+       resize a view with copying old data to new data at the corresponding indices
+
+    .. cppkokkos:function:: realloc(const size_t n0 = 0, const size_t n1 = 0, const size_t n2 = 0, const size_t n3 = 0, const size_t n4 = 0, const size_t n5 = 0, const size_t n6 = 0, const size_t n7 = 0)
+
+       resize a view with discarding old data
+
+
+    .. rubric:: *Private* Members
+
+    :member: typedef original_view_type internal_view_type;
+    :member: internal_view_type internal_view;
+
+
+.. rubric:: Free Functions
+
+.. cppkokkos:function:: contribute(View<DT1, VP...>& dest, Kokkos::Experimental::ScatterView<DT2, LY, ES, OP, CT, DP> const& src)
+
```

Comment:
Should the `parallel_reduce` info at the top precede the `contribute` function?

Reply:
no, par_red at the top is just a link